recipes/kmetashot/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name    = "kmetashot" %}
+{% set version = "0.0.0_master" %}
+{% set version_py = "0.0.0+master" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/gdefazio/kMetaShot/archive/refs/heads/master.zip
```

Comment:
please do not use master branches. Can you please create a proper github release?